db/migrations/20200504135835_remove_urn_state.sql

```diff
@@ -0,0 +1,226 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+DROP FUNCTION IF EXISTS api.all_urn_states;
+DROP FUNCTION IF EXISTS api.get_urn;
+
+DROP FUNCTION IF EXISTS api.bite_event_urn;
+CREATE OR REPLACE FUNCTION api.bite_event_urn(event api.bite_event) RETURNS api.urn_snapshot
```

Comment:
Do we need the `DROP FUNCTION` if we're doing `CREATE OR REPLACE`?

Reply:
Unfortunately since we change the signature of the function, we have to drop it first. You can't change the return value with a replace.

We probably don't need `CREATE OR REPLACE` - I was being overly-cautious..